presto-docs/src/main/sphinx/functions.rst

```diff
@@ -38,3 +41,4 @@ statement and syntax reference</sql>` are available.
     functions/color
     functions/session
     functions/teradata
+    functions/list
```

Comment:
Having this here is strange, since all of the rest of the items are for specific types of functions. Let's create a new top level section and put this under `index/functions`.

Reply:
Not sure how you would link that together .. have a second index page? It has to be inserted in the ToC tree somewhere .. so make a new toctree like in release?

Reply:
That’s what I was thinking.

Reply:
I am not sure how this would fit together... since we already have a /index.rst ... make .index/index and index/functions ?